docs/telemetry.mdx

```diff
@@ -109,4 +111,12 @@ If you are using `@clerk/clerk-react` directly, or using an SDK that doesn't hav
   ```env {{ filename: '.env.local' }}
   PUBLIC_CLERK_TELEMETRY_DISABLED=1
   ```
+
+  ```env {{ filename: '.env' }}
+  CLERK_TELEMETRY_DISABLED=1
+  ```
+
+  ```env {{ filename: '.env' }}
+  VITE_CLERK_TELEMETRY_DISABLED=1
+  ```
```

Comment:
did you put these in the wrong order? I believe the `vite` one is meant for React Router

Reply:
`VITE_` was missing, fixed